Snakefile

```diff
@@ -77,8 +43,8 @@ rule filter:
           - minimum genome length of {params.min_length}
         """
     input:
-        sequences = rules.parse.output.sequences,
-        metadata = rules.parse.output.metadata,
+        sequences = S3.remote(BUCKET + "sequences.fasta"),
+        metadata = S3.remote(BUCKET + "metadata.tsv"),
```

Comment:
Probably still useful to have a `download` rule which only copies the files from S3 to the local disk under `data/` so that the files can be inspected more easily by whoever is running the pipeline. This would then leave the rest of the rules unchanged, make it easier to commit the updated metadata.tsv to this repo, and allow someone who doesn't have access to the bucket to drop files into place and still run the build.

Reply:
I added back a download rule that downloads fasta and metadata files in one rule. Should this rule use the `aws` command?

Reply:
Oh, it doesn't need to. For example:

```
rule download:
    input: S3.remote("foo/bar/baz.txt")
    output: "data/baz.txt"
    shell: "cp {input:q} {output:q}"
```

Reply:
If doing that, the `stay_on_remote=True` option to `S3RemoteProvider` can be useful.

Reply:
When I use the `stay_on_remote=True` optoin, Snakemake is unable to fin the file on S3 in my copy command.

Reply:
Ah, yeah, I see. Looking at the source, that feature definitely doesn't work as it is documented to.